templates/validator/charts.html

```diff
@@ -242,7 +236,8 @@
                                 const startEpoch = timeToEpoch(tooltip.chart.hoverPoint.x)
                                 const timeForOneDay = 24 * 60* 60 * 1000
                                 const endEpoch = timeToEpoch(tooltip.chart.hoverPoint.x + timeForOneDay) - 1
-                                text += `${new Date(tooltip.chart.hoverPoints[0].x).toLocaleDateString()} (Epochs ${startEpoch} - ${endEpoch})<br/>`;
+                                const endTs = epochToTime(endEpoch)
+                                text += `${new Date(tooltip.chart.hoverPoints[0].x).toLocaleDateString()} - ${new Date(endTs).toLocaleDateString()} <br> Epochs ${startEpoch} - ${endEpoch}<br/>`;
```

Comment:
**issue:** According to the description/comments we do not just want to show the date in the tooltip but the exact time for the network.

Reply:
fixed